README.md

```diff
@@ -26,13 +26,25 @@
 
 The friendly Operating System for IoT!
 
+RIOT is a real-time multi-threading operating system that supports a range of
+devices that are typically found in the Internet of Things (IoT): 
+8-bit microcontrollers, 16-bit microcontrollers and light-weight 32-bit processors.
```

Comment:
I think both is okay (in fact, we're talking about the word size of the processor here), but I agree that it should be consistent.